MosaicSlider/MosaicSlider.js

```diff
@@ -4,16 +4,29 @@ import { draw_primitive } from "../sketchlib/draw_primitive.js";
 import { fix_mouse_coords } from "../sketchlib/fix_mouse_coords.js";
 import { InteractiveMosaic } from "./InteractiveMosaic.js";
 import { prevent_mobile_scroll } from "../sketchlib/prevent_mobile_scroll.js";
+import { SoundManager } from "./SoundManager.js";
 
 //const INITIAL_COLORS = ["#231f20", "#bb4430", "#7ebdc2", "#f3dfa2"];
 // clouds, sky, grass, dirt
 const INITIAL_COLORS = ["#ccf0ef", "#5697d8", "#456538", "#633912"];
 
+const SOUND_MANIFEST = {
+  sfx: {
+    whish: {
+      url: "./sounds/whish.wav",
+      volume: -9,
+    },
+  },
+};
+
+//@ts-ignore
+const SOUND = new SoundManager(Tone, SOUND_MANIFEST);
```

Comment:
Adding a volume toggle button in the top right would be helpful so the user can opt-out if desired.

This and the play overlay will be helpful for other sketches like #26